nx/scripts/nexter.js

```diff
@@ -116,12 +116,15 @@ function decorateLinks(el) {
 
 function decorateSections(parent, isDoc) {
   const selector = isDoc ? 'main > div' : ':scope > div';
-  return [...parent.querySelectorAll(selector)].map((el) => {
+  return [...parent.querySelectorAll(selector)].map((el, i) => {
     el.classList.add('section');
     el.dataset.status = 'decorated';
     el.autoBlocks = decorateLinks(el);
     el.blocks = [...el.querySelectorAll(':scope > div[class]')];
     decorateDefaults(el);
+    if (i === 0 && sampleRUM.enhance) {
```

Comment:
What does `sampleRUM.enhance` do? We should not have a dependency on rum for basic decorating. It provides the user trying to access the page no value.

Reply:
`enhance` is an optimization to load the enhancer only after the first section is loaded, i.e. load the enhancer post LCP.

Reply:
If we lazy load `sampleRUM`, this would not be necessary.